commit bcc04b063a8a9e933479d030f59d9a04750032ae
Author: Petr Danecek <pd3@sanger.ac.uk>
Date:   Sun Jun 15 18:11:16 2025 +0200

    Update Makefile for scripts to be included in bioconda recipes. Resolves #2375

diff --git a/Makefile b/Makefile
index 0b936b2c..1b5ff85f 100644
--- a/Makefile
+++ b/Makefile
@@ -71,12 +71,15 @@ INSTALL_SCRIPT  = $(INSTALL_PROGRAM)
 PROGRAMS = bcftools
 MISC_SCRIPTS = \
     misc/color-chrs.pl \
+    misc/gff2gff \
     misc/gff2gff.py \
     misc/guess-ploidy.py \
     misc/plot-vcfstats \
     misc/plot-roh.py \
+    misc/roh-viz \
     misc/run-roh.pl \
-    misc/vcfutils.pl
+    misc/vcfutils.pl \
+    misc/vrfs-variances
 TEST_PROGRAMS = test/test-rbuf test/test-regidx
 
 ALL_CPPFLAGS = -I. $(HTSLIB_CPPFLAGS) $(CPPFLAGS)
